sphinx/ext/autosummary/generate.py

```diff
@@ -239,15 +239,21 @@ def skip_member(obj: Any, name: str, objtype: str) -> bool:
                            name, exc, type='autosummary')
             return False
 
+    def attr_getter(obj, name, *defargs):
+        return sphinx.ext.autodoc.autodoc_attrgetter(app, obj, name, *defargs)
+
+    def get_all_members(obj: Any) -> Dict[str, ObjectMember]:
+        all_members = sphinx.ext.autodoc.get_class_members(obj, [qualname], attr_getter)
+        return all_members
+
     def get_members(obj: Any, types: Set[str], include_public: List[str] = [],
                     imported: bool = True) -> Tuple[List[str], List[str]]:
         items: List[str] = []
         public: List[str] = []
-        for name in dir(obj):
-            try:
-                value = safe_getattr(obj, name)
-            except AttributeError:
-                continue
+
+        all_members = get_all_members(obj)
```

Comment:
`get_members()` is used for both modules and classes. But `get_all_members()` is implemented only for classes. Is this working correctly?